exporter/exporter.go

```diff
@@ -324,23 +327,34 @@ func (exporter *NebulaExporter) CollectMetrics(
 	namespace string,
 	metrics []string,
 	ch chan<- prometheus.Metric) {
-	for _, singleMetric := range metrics {
-		seps := strings.Split(singleMetric, "=")
-		if len(seps) != 2 {
-			continue
-		}
-
+	// status metric
+	if len(metrics) == 1 {
+		seps := strings.Split(metrics[0], "=")
 		values, err := strconv.ParseFloat(seps[1], 64)
-		if err != nil {
-			continue
+		if err == nil {
+			if metric, ok := exporter.metricMap[seps[0]]; ok {
+				ch <- prometheus.MustNewConstMetric(metric.desc, prometheus.GaugeValue, values,
+					name,
+					namespace,
+					componentType,
+				)
+			}
 		}
-
-		if metric, ok := exporter.metricMap[seps[0]]; ok {
-			ch <- prometheus.MustNewConstMetric(metric.desc, prometheus.GaugeValue, values,
-				name,
-				namespace,
-				componentType,
-			)
+	} else {
+		// query metrics
+		matches := convertToMap(metrics)
+		for metric, value := range matches {
+			values, err := strconv.ParseFloat(value, 64)
```

Comment:
values feel like an array or slice, use value or v ?

Reply:
right, variable name "v" is better